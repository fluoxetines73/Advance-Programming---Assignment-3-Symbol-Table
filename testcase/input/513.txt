INSERT b string
LOOKUP b
ASSIGN b 'HelloWorld'