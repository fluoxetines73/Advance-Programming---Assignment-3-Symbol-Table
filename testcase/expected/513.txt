success
0
success